dxr/plugins/xpidl/__init__.py

```diff
@@ -0,0 +1,28 @@
+from functools import partial
+from os.path import abspath
+
+from schema import Optional, Use, And
+
+from dxr.config import AbsPath
+from dxr.plugins import Plugin, AdHocTreeToIndex, filters_from_namespace
+from dxr.plugins.xpidl import filters
+from dxr.plugins.xpidl.mappings import mappings
+from dxr.plugins.xpidl.indexers import FileToIndex
+
+
+def split_on_colon_into_abspaths(value):
+    return map(abspath, value.strip().split(':'))
+
+
+ColonPathList = And(basestring,
```

Comment:
Why make up a new thing when you can just do what ignore_patterns does? (See the explanation in configuration.rst.)